pinax/stripe/webhooks.py

```diff
@@ -79,11 +79,11 @@ def validate(self):
         """
         Validate incoming events.
 
-        We fetch the event data to ensure it's legit. For Connect
-        accounts we must fetch the event using the `stripe_account`
-        parameter, else we won't find it.
+        We fetch the event data to ensure it is legit.
+        For Connect accounts we must fetch the event using the `stripe_account`
+        parameter.
         """
-        self.stripe_account = self.event.webhook_message.get("user_id")
+        self.stripe_account = self.event.webhook_message.get("account")
```

Comment:
Does this still work for non-connect accounts?

Reply:
I guess so, It will just flow through as `None`.

Reply:
I confirm